src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ET1ServingServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.et.common.model.ccd.items.DocumentTypeItem;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.ET1ServingService;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+class ET1ServingServiceTest {
+    private static CaseDetails caseDetails;
+
+    private static uk.gov.hmcts.ethos.replacement.docmosis.service.ET1ServingService ET1ServingService;
+
+    @BeforeAll
+    static void setUp() throws Exception {
+        caseDetails = generateCaseDetails("midServingCaseDetailsTest.json");
+        ET1ServingService = new ET1ServingService();
+    }
+
+    private static CaseDetails generateCaseDetails(String jsonFileName) throws Exception {
+        String json = new String(Files.readAllBytes(Paths.get(Objects.requireNonNull(
+                ET1ServingServiceTest.class.getClassLoader()
+                .getResource(jsonFileName)).toURI())));
+        ObjectMapper mapper = new ObjectMapper();
+        return mapper.readValue(json, CaseDetails.class);
+    }
+
+    @Test
+    void generateOtherTypeDocumentName() {
+        String expectedDocumentName = "**<big>test-filename.xlsx</big>**<br/><small>Test description</small><br/>";
+        List<DocumentTypeItem> documentTypeItems = caseDetails.getCaseData().getServingDocumentCollection();
+        assertThat(expectedDocumentName, is(ET1ServingService.
+                generateOtherTypeDocumentName(documentTypeItems)));
+    }
+
+    @Test
+    void generateClaimantAndRespondentAddress() {
+        String expectedClaimantAndRespondentAddress = "**<big>Claimant</big>**<br/>Doris Johnson"
+                + "<br/>232 Petticoat Square<br/>22 House<br/>London<br/>W10 4AG<br/><br/>"
+                + "**<big>Respondent 1</big>**<br/>Antonio Vazquez"
+                + "<br/>11 Small Street<br/>22 House<br/>Manchester<br/>M12 42R<br/><br/>"
+                + "**<big>Respondent 2</big>**<br/>Juan Garcia<br/>12 Small Street<br/>24 House"
+                + "<br/>Manchester<br/>M12 4ED<br/><br/>";
+
+        CaseData caseData = caseDetails.getCaseData();
+        assertThat(expectedClaimantAndRespondentAddress,
+                is(ET1ServingService.generateClaimantAndRespondentAddress(caseData)));

```

Comment:
I think you may have this the wrong way round.  I.e. should it not be:
assertThat(ET1ServingService.generateClaimantAndRespondentAddress(caseData),
                is(expectedClaimantAndRespondentAddress));

If I'm right, best fix this one and in other places too.
Also, as a slight preference in future, it's a little more readable using Hamcrest's way of assertThat().is...()